docs/README.md

```diff
@@ -460,22 +460,28 @@ export class Post {
 }
 ```
 
-Entities represent the internal state of our system and can be queried from command or
+Entities represent our domain model and can be queried from command or
 event handlers to make business decisions or enforcing business rules.
 
 #### 5. First read model
 
-In a real application, we rarely want to publish our entire domain model (entities)
-including all their fields. Also, different people may access one or other data depending
+In a real application, we rarely want to make public our entire domain model (entities)
+including all their fields. What is more, different users may have different views of the data depending
```

Comment:
I usually see `in addition` or `moreover` , but feel free to leave your version with `What is more` (baby don't hurt me, don't hurt me, no more 🎶 )

Reply:
JAJAJAJAJAJAJAJAJA